pkg/k8s/client.go

```diff
@@ -700,7 +700,7 @@ func (c *Client) Label(obj runtime.Object, labels map[string]string) error {
 	return nil
 }
 
-func (c *Client) CreateOrUpdateNamespace(name string, labels, annotations map[string]string) error {
+func (c *Client) CreateOrUpdateNamespace(name string, labels, annotations map[string]string, finalizers []string) error {
```

Comment:
Why do we need this?

Reply:
Added it to create the namespace being tested,
but we can do that test manually.